src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -299,6 +299,53 @@ public virtual async Task PatchOne_Update_Test()
                 );
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a computed field is missing from the request body.
+        /// </summary>
+        [TestMethod]
+        public virtual async Task PatchOneWithComputedFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PATCH update when a computed field (here 'last_sold_on_date')
+            // is missing from the request body.
+            string requestBody = @"
+            {
+                ""book_name"": ""New book"",
+                ""copies_sold"": 50,
+                ""last_sold_on"": null
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PatchOneUpdateWithComputedFieldMissingFromRequestBody"),
+                    operationType: EntityActionOperation.UpsertIncremental,
+                    requestBody: requestBody,
+                    expectedStatusCode: HttpStatusCode.OK
+                );
+
+            // Validate successful execution of a PATCH insert when a computed field (here 'last_sold_on_date')
+            // is missing from the request body.
+            requestBody = @"
+            {
+                ""book_name"": ""New book"",
+                ""copies_sold"": 50
+            }";
```

Comment:
What's the significance of including/excluding `last_sold_on` in the PATCH update versus insert?

`last_sold_on_date` is the auto generated column.

Should be described in the test summary.

Reply:
There is no difference. Just validating the changes for insert/update via PATCH. Updated the comment and the request body for the update test.